scss/_custom-forms.scss

```diff
@@ -164,6 +164,11 @@
     }
   }
 
+  &[multiple] {
+    height: auto;
+    background: none;
```

Comment:
Changing this to `background-image` would be helpful so we don't lose the `background-color` set above.